google/cloud/pubsub/internal/tracing_pull_lease_manager.cc

```diff
@@ -79,6 +86,8 @@ class TracingPullLeaseManager : public PullLeaseManager {
   }
 
   std::shared_ptr<PullLeaseManager> child_;
+  opentelemetry::trace::SpanContext consumer_span_context_ =
+      opentelemetry::trace::SpanContext::GetInvalid();
```

Comment:
I am not sure you need an initializer here if the constructor always initialize the member variable.

Reply:
I think I added this since there is no default constructor for SpanContext()

Reply:
But there is no default constructor for `TracingPullLeaseManager` either?  No matter.